package.json

```diff
@@ -11,47 +11,75 @@
     "umd"
   ],
   "scripts": {
-    "build": "nwb build-react-component",
-    "clean": "nwb clean-module && nwb clean-demo",
-    "lint": "npx eslint ./",
+    "build": "npm run build:umd && npm run build:demo",
+    "build:demo": "NODE_ENV=development webpack --mode=development",
+    "build:umd": "NODE_ENV=production webpack --mode=production",
+    "build:es": "mkdir -p es && cp -r src/* es && NODE_ENV=production MODULE_FORMAT=es npx babel es -d es",
+    "build:cjs": "mkdir -p lib && cp -r src/* lib && NODE_ENV=production MODULE_FORMAT=cjs npx babel lib -d lib",
+    "parcel": "parcel demo/src/index.html",
+    "lint": "eslint ./src ./__tests__",
     "prepublishOnly": "npm run build",
```

Comment:
This would currently only provide the `umd` build. I'm not sure what people expect to find in the npm package. Should we change this to
```
"prepublishOnly": "npm run build:es && npm run build:cjs && npm run build"
```
or even
```
"prepublishOnly": "npm run clean && npm run build:es && npm run build:cjs && npm run build"
```
and add a `clean` task that removes all builds including the demo build? This would be more harmonized with the core repo.